manifests/Captura/9.0.0-beta4.yaml

```diff
@@ -0,0 +1,14 @@
+Id: MathewSachin.Captura
+Version: 9.0.0-beta4
+Name: Captura
+Publisher: MathewSachin
+License: MIT
+LicenseUrl: https://github.com/MathewSachin/Captura/blob/master/LICENSE.md
+AppMoniker: captura
+Description: Capture Screen, WebCam, Audio, Cursor, Mouse Clicks and Keystrokes
+Homepage: https://mathewsachin.github.io/Captura
+Installers:
+  - Arch: x86
+    Url: https://github.com/MathewSachin/Captura/releases/download/v8.0.0/Captura-Setup.exe
```

Comment:
Version in download URL does not match version specified above.

Reply:
Updated link